### ISSUE TITLE
Question: is there a way to keep using the latest version of cmder on windows 7?

### ISSUE THREAD

**Issue by candrapersada** [cmderdev/cmder#2949]
### Question

> In the next major releases of Cmder, the minimum Windows requirement will be bumped from Windows 7 SP1 to Windows 10, as well.

Is there a way to still be able to use the latest version of cmder on windows 7?

### Checklist

- [X] I have read the documentation and made sure what I'm looking for isn't present, or is unclear.
- [X] I have searched for similar issues, and either this question wasn't asked before, or I didn't find any that describe my question.

**Comment by DRSDavidSoft** [comment]
Yes, we will keep supporting Windows 7 SP1 as long as our dependencies continue to support it;

Specifically, ConEmu, Clink, and git-for-windows.

After that happens, you can keep using the last known version of Cmder to support Windows 7 and even re-download an old version, if required.

If we drop ConEmu in favor of Windows Terminal, I believe bumping the minimum supported OS to Windows 10 will be a necessity.

In any case, Windows 10 supports _almost_ all the hardware that 7 supports, so it's recommend to upgrade when you can. Windows 7 has not been been getting any security updates in years so it's unadvisable to continue using unsupported software.
